mm2src/mm2_main/src/lp_wallet.rs

```diff
@@ -499,3 +500,87 @@ pub async fn get_mnemonic_rpc(ctx: MmArc, req: GetMnemonicRequest) -> MmResult<G
         },
     }
 }
+
+/// `GetWalletError` is an enum representing possible errors that might occur during the `get_wallet` RPC method.
+///
+/// It includes variants for situations where the wallet name is not initialized or there is an internal error.
+#[derive(Debug, Display, Serialize, SerializeErrorType)]
+#[serde(tag = "error_type", content = "error_data")]
+pub enum GetWalletError {
+    /// The wallet name has not been initialized in the context.
+    #[display(fmt = "Wallet name is not initialized")]
+    WalletNameNotInitialized,
+
+    /// An internal error occurred while processing the request.
+    #[display(fmt = "Internal error: {}", _0)]
+    Internal(String),
+}
+
+impl HttpStatusCode for GetWalletError {
+    fn status_code(&self) -> StatusCode {
+        match self {
+            GetWalletError::WalletNameNotInitialized => StatusCode::BAD_REQUEST,
+            GetWalletError::Internal(_) => StatusCode::INTERNAL_SERVER_ERROR,
+        }
+    }
+}
+
+/// `GetWalletRequest` is a struct representing the parameters for the `get_wallet` RPC method.
+///
+/// Currently, it does not contain any fields but serves as a placeholder for future extensibility.
+/// It ensures compatibility with future versions that might introduce new fields.
+#[derive(Debug, Deserialize, Default)]
+#[serde(default)]
+pub struct GetWalletRequest {
+    // Currently empty, but may hold future parameters.
+}
+
+/// `GetWalletResponse` is a struct representing the response from the `get_wallet` RPC method.
+///
+/// It currently includes the name of the active wallet. This struct is designed to be extendable in the future
+/// to include more wallet properties such as balance, state, or configurations.
+#[derive(Debug, Serialize)]
+pub struct GetWalletResponse {
+    /// The name of the currently active wallet.
+    pub wallet_name: String,
+}
+
+/// Retrieves the active wallet name from the context.
+///
+/// # Parameters
+///
+/// - `ctx`: The context.
+/// - `req`: The request structure (optional).
+///
+/// # Returns
+///
+/// A `Result` type containing:
+///
+/// * `Ok(GetWalletResponse)` - The wallet name.
+/// * `MmError<GetWalletError>` - Error indicating the wallet name is not initialized.
```

Comment:
there is no need to add params and return in doc comment for `get_wallet_rpc` function

we prefer to add doc comms right above the params and return types: GetWalletRequest, GetWalletResponse, GetWalletError

so you can remove Parameters and Returns and Examples :)

Reply:
I guess @CharlVS was following the `get_mnemonic_rpc` example, I was trying different things back then when I implemented it. I wanted also to have request/response examples in docs e.g.
https://github.com/KomodoPlatform/komodo-defi-framework/blob/7723b50fcb533ebe12b621eb862b8a958113696b/mm2src/mm2_main/src/lp_wallet.rs#L330-L357 But I guess API docs will be moved from docs repo to KDF and we won't need this. Thanks for trying to keep docs consistent @laruh we need to have a consistent way of writing docs across all modules. I am working on this implementation in a different branch anyway and will be opening another PR this week, just wanted to clarify this point here :)